src/setup.ts

```diff
@@ -6,13 +6,13 @@ const setup = () => {
         for (let y = 0; y < board[x].length; y++) {
             const square = document.createElement('div');
             square.id = `${x},${y}`;
-            square.innerHTML = board[x][y] ? board[x][y].display : '';
+            square.innerHTML = board[x][y] ? board[x][y]!.display : '';
             square.className = 'square';
             square.className += x % 2 == y % 2 ? ' light' : ' dark';
             square?.addEventListener('click', (e) => {
                 touched(e);
             });
-            document.getElementById('board')?.appendChild(square);
```

Comment:
Raczej stosuje się "?" zamiast !

Reply:
Ok poprawione, zamieniłem znaki zapytania bo myślałem że oznaczają one opcjonalną wartość a znak "!" - non nullable/undefined

Reply:
? mówi nam jeżeli nie jest undefinded to idx dalej